test/from-draft7/to-2019-09/dependencies-to-dependentRequired.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "title": "`dependencies` has properties whose subschema has type array",
+        "from": {
+            "dependencies": {
+                "foo": [ "bar", "baz" ],
+                "car": [ "wheel", "foo" ]
+            },
+            "dependentRequired": {}
```

Comment:
Huh.. this is to initialize an dempty dependentRequired object, so that we may fill it later..
Thats because we cannot put both in a single rule, that will be like saying to re-initialize the multiple times So I changed it

Reply:
> Thats because we cannot put both in a single rule, that will be like saying to re-initialize the multiple times So I changed it

Not sure I understand why this is a blocker. Remember that a test doesn't need to (and it is not expected to) exercise only one rule.

Reply:
@jviotti I have updated this